README.md

```diff
@@ -42,55 +42,62 @@ Write here a short description of your benchmark.
-   1. Fill the "Repository name" field with a name for your controller.
-   1. Choose the visibility of your controller, keep it "Public" if you don't care about people looking at your controller code otherwise set it to "Private".
-   1. Finally, click on the green button "Create repository from template".
+### 2. Add the Organizer as a Collaborator
 
-You should now be on your **own** repository page and not the benchmark creator's repo. **This is important** in order to be able to use the links in the following chapters.
+You can skip this step if you created your repository as "Public" instead of "Private".
 
-#### If you set your repository as private, add the organizer as collaborator
-
-1. [Click here](../../settings/access) to go to the "Collaborators" setting page
-   1. You might need to confirm the access by re-entering your GitHub password.
-1. You should see a "Manage access" box where you will see the current collaborators of the repo.
+- [Click here](../../settings/access) to go to the "Collaborators" setting page. You might need to confirm the access by re-entering your GitHub password.
+- You should see a "Manage access" box where you will see the current collaborators of the repo.
 Click on the "Add people" and search for "ORGANIZER_NAME". When you found the organizer, add them to the repository.
 
-### Register by posting an issue
+### 3. Setup your Secret Token
+
+In order to allow your repository to automatically trigger evaluations of your robot controller when you push modifications to the main branch, you will have to setup a secret token:
+
+- [Create a new Personal Access Token](../../../../settings/tokens/new). Give it a name referring to the benchmark to remember what it is for and set its "Expiration" to six months or so. You can always set it to "No expiration" or recreate a token when it expires to allow the automated scripts to continue working. Tick the "public_repo" scope box and scroll down to click on the "Generate token" button. Copy the generated code to your clipboard.
+- Go to the repo's [secrets settings](../../settings/secrets/actions/new) to create a new repository secret. Name it "TRIGGER_TOKEN". In the "Secret" text area, paste the Personal Access Token you just created and finally click the "Add secret" button.
+
+### 4. Modify your Robot Controller
+
+You can now modify your [main robot controller file](../../edit/main/controllers/edit_me/edit_me.py) or create new files in this folder and push the modification to the main branch of your repository.
+A series of automated actions will take place in a few seconds.
+If everything went well, your repository should appear after some time in the [leader board](https://benchmark.webots.cloud/run?version=R2022b&url=https%3A%2F%2Fgithub.com%2Fcyberbotics%2Fbenchmark-template%2Fblob%2Fmain%2Fworlds%2Frobot_programming.wbt&type=benchmark) of the benchmark.
```

Comment:
The link to the leader board will therefore need to be updated at two different places, right ?

Maybe we could use a reference-style link to set the link in one place and use it at multiple spots easily: https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
Or maybe we could tell the participants that they can click on the shield badge at the top of the readme to get to the leader board